src/scripts/main.js

```diff
@@ -1,3 +1,263 @@
 'use strict';
 
-// write code here
+const minNameLength = 4;
+const minAge = 18;
+const maxAge = 90;
+
+const table = document.querySelector('table');
+const ths = table.querySelectorAll('thead th');
+let tbody = table.querySelector('tbody');
+let rows = Array.from(tbody.querySelectorAll('tr'));
+
+// создаем форму
```

Comment:
it is bad practice to make comments not in English
consider to practice comment code in English